pkg/kubelet/config/flags.go

```diff
@@ -65,6 +65,20 @@ type ContainerRuntimeOptions struct {
 	// CNICacheDir is the full path of the directory in which CNI should store
 	// cache files
 	CNICacheDir string
+
+	// Image credential provider plugin options
```

Comment:
thinking about @liggitt comment: re relative to root-dir.

if it were relative, i would assume we would not have to expose a flag at all in favor of prescribed default.

i do not think we know enough yet to say if that makes sense as its not clear to me how cloud vendors may configure their default os image building one way or the other, and that varies with each cloud.  as a result, a separate flag seems fine for moment.